### ISSUE TITLE
Mnesia demo partially broken

### ISSUE THREAD

**Issue by oubiwann** [lfe/lfe#397]
I've had to comment out the function `by_place_qlc` in `examples/mnesia_demo.lfe` due to the following error:

```lisp
lfe> (c "examples/mnesia_demo.lfe")
examples/mnesia_demo.lfe:90: symbol -rec-0- unbound
examples/mnesia_demo.lfe:90: symbol -rec-0- unbound
error
```

Note line numbers are from a branch, not `develop`.

**Comment by rvirding** [comment]
This now gets a different error, which of course needs fixing.

**Comment by oubiwann** [comment]
Progress!

**Comment by oubiwann** [comment]
Yeah, just did a quick check on this myself, and see the change in error you mentioned. It's now this:

```lisp
lfe> (c "examples/mnesia_demo.lfe")
examples/mnesia_demo.lfe:88: function #(emp-person 2) undefined
error
```

**Comment by oubiwann** [comment]
@rvirding any progress on this one?

If not, I'd be okay with marking this for LFE 2.1 ...

**Comment by oubiwann** [comment]
I've updated the example as part of a big example cleanup, thanks to @diasbruno's diligence as he explores LFE 😄. When I open the PR, I'll mention this ticket as closable after merge.

**Comment by oubiwann** [comment]
Closing in favour of the bigger, new ticket.
